Makefile

```diff
@@ -12,5 +26,11 @@ updaterdk:
 	go get go.viam.com/rdk@latest
 	go mod tidy
 
+FFmpeg:
+	git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
```

Comment:
Mind please putting in the README that this will use ffmpeg 6.1 release?

Reply:
sure thing - added small section to the README explaining FFmpeg version.